601-spring-data-primitive-types/src/test/java/io/quarkus/qe/spring/data/BookResourceTest.java

```diff
@@ -48,11 +54,44 @@ void testCustomFindPublicationIsbnObjectLong() {
     // QUARKUS-525
     @Test
     void testFindBooksByPublisherZipCode() {
+        List<Book> books = retrieveBooksByZipcode();
+        books.stream().forEach(book -> assertThat(book.getPublisherAddress().getZipCode(), is("28080")));
+    }
+
+    // https://github.com/quarkusio/quarkus/issues/13234
+    @Test
+    @Disabled
+    void testJpaFieldsMapping() {
+        Book book = retrieveBooksByZipcode().stream().findFirst().get();
+
+        // Post a new comment on an existing book
+        Map<String, String> comment = Collections.singletonMap("Shakespeare", "Lorem Ipsum Lorem Ipsum");
+        book.setComments(comment);
+        book = updateBook(book);
+
+        assertThat(book.getComments().size(), equalTo(1));
+        assertThat(book.getComments(), equalTo(comment));
+
+        // Update the previous comment
+        Map<String, String> commentUpdated = Collections.singletonMap("Shakespeare", "Lorem Ipsum Lorem Ipsum novus");
+        book.setComments(comment);
```

Comment:
If I understand it right, here should be the updated comment.
```suggestion
        book.setComments(commentUpdated);
```

Reply:
Hi @jsmrcka, yes this is the cause of the test failure